mautic/mautic
# Allow html to textarea custom field

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR bring option to allow HTML in  text area custom contact/company field.
We have many request store HTML to contact and then work with this content.

![image](https://user-images.githubusercontent.com/462477/56963867-4035b100-6b5a-11e9-9915-9eb46a241394.png)

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create textarea contact field and enable allow HTML
3. Create/edit contact and see If this textarea work also with HTML

![image](https://user-images.githubusercontent.com/462477/56963915-622f3380-6b5a-11e9-8fb8-29731fa2096f.png)
